### PR TITLE
docs: update references to ext. integrations

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

drop archived projects and link the overview from external source

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

----
📚 Documentation preview 📚: https://pytorch-lightning--19248.org.readthedocs.build/en/19248/

cc @borda @carmocca

### CODE REVIEW

**Comment by Borda** [comment]
updating also HPU as it was missed twice, ref https://github.com/Lightning-AI/lightning-Habana/pull/138

File: docs/source-pytorch/advanced/model_parallel.rst [hunk 1]
```diff
@@ -117,41 +117,4 @@ Third-party strategies
 **********************
 
 Cutting-edge Lightning strategies are being developed by third-parties outside of Lightning.
-If you want to try some of the latest and greatest features for model-parallel training, check out these integrations:
-
-.. raw:: html
```
Comment (Borda) [inline docs/source-pytorch/advanced/model_parallel.rst:122]: this is moved to own index page

File: docs/source-pytorch/conf.py [hunk 2]
```diff
@@ -107,6 +107,13 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     os.makedirs(os.path.dirname(img_), exist_ok=True)
     urllib.request.urlretrieve(f"{URL_RAW_DOCS_GRAPHCORE}/{img}", img_)
 
+# Copy strategies docs as single pages
+assist_local.AssistantCLI.pull_docs_files(
+    gh_user_repo="Lightning-Universe/lightning-Hivemind",
+    target_dir="docs/source-pytorch/integrations/strategies",
+    checkout="7b985855f08c4ff94299c06f912f16b64ee93888",  # Todo: replace with SHA on master after the related PR is merged
```
Comment (Borda) [inline docs/source-pytorch/conf.py:114]: This need to be updated before merging

**Review by lantiga — APPROVED** [review]

File: docs/source-pytorch/extensions/strategy.rst [hunk 3]
```diff
@@ -113,12 +113,6 @@ There are powerful third-party strategies that integrate well with Lightning but
    * - Name
      - Package
      - Description
-   * - ColossalAI
```
Comment (awaelchli) [inline docs/source-pytorch/extensions/strategy.rst:116]: This table now contains only one row. I think you can just delete it and do the same as above, linking to the new page.

**Review by awaelchli — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]

**Pull request merged**